src/egon/data/datasets/gas_grid.py

```diff
@@ -1034,7 +1094,7 @@ class GasNodesAndPipes(Dataset):
     #:
     version: str = "0.0.9"

```

Comment:
You should update the version number

Reply:
Done

Reply:
Done in 489eebeed1513adf8cb08570803cbe2f301566da